lib/killbill_client/models/payment.rb

```diff
@@ -12,16 +12,27 @@ class Payment < PaymentAttributes
       has_custom_fields KILLBILL_API_PAYMENTS_PREFIX, :payment_id
 
       class << self
-        def find_by_id(payment_id, options = {})
+        def find_by_id(payment_id, with_plugin_info = false, options = {})
```

Comment:
Another option here, would be to expect a `with_plugin_info` in the options hash, but I'm trying to keep the code some what consistent with other usages of with_plugin_info:
https://github.com/killbill/killbill-client-ruby/blob/master/lib/killbill_client/models/invoice_payment.rb#L10
https://github.com/killbill/killbill-client-ruby/blob/master/lib/killbill_client/models/payment_method.rb#L10